### DISCUSSION TITLE
Resource grouping

### DISCUSSION THREAD

**Discussion by stwebage** [vkurko/calendar#172] [Q&A]
There are tasks when it is necessary to group a large number of resources. Let's say I have a calendar in which I enter data on renting courts for playing: tennis, table tennis and badminton. I have 5 of each type of sites and it turns out that I have 15 columns in the calendar, which turns out to be an unreadable option, I also don’t want to make a separate calendar for each type, can there be an opportunity to group resources?

**Comment by vkurko** [comment]
Perhaps in your case you should arrange resources in rows, not columns. Unfortunately, such a view has [not yet been implemented](https://github.com/vkurko/calendar/issues/30).

**Reply by Rasanoob** [reply]
In my case I have a comparable issue. Is there a way to hide resources and just show the one the user selects? There is an option to hide empty resources [filterResourcesWithEvents](https://github.com/vkurko/calendar#filterresourceswithevents) so I'm wondering if there is an option to show a selection. Otherwise I would show just one resource and load another one by a custom javascript event listener.

**Reply by vkurko** [reply]
There is no _option to show a selection_, so you must implement such a widget yourself. And you can manage the displayed resources by overriding the list of resources like this `calendar.setOption('resources', newResourcesList)`.

**Reply by Rasanoob** [reply]
Thanks for your answer. Should be doable.
